test/file_collection_tests.coffee

```diff
@@ -301,10 +301,14 @@ if Meteor.isServer
               test.fail(err) if err
               testColl.importFile testfile, {}, bind_env (err, doc) ->
                 test.fail(err) if err
-                readstream = testColl.findOneStream {_id: doc._id }
-                readstream.on 'data', bind_env (chunk) ->
-                  test.equal chunk.toString(), '1234567890','Incorrect data read back from file stream'
-                readstream.on 'end', bind_env () ->
+                test.equal typeof doc, 'object', "No document imported"
+                if typeof doc == 'object'
+                  readstream = testColl.findOneStream {_id: doc._id }
+                  readstream.on 'data', bind_env (chunk) ->
+                    test.equal chunk.toString(), '1234567890','Incorrect data read back from file stream'
+                  readstream.on 'end', bind_env () ->
+                    onComplete()
+                else
                   onComplete()
```

Comment:
I appreciate you bringing this to my attention. These tests pre-date Meteor's Windows support. I'd prefer to actually fix the test to do something reasonable under Windows (rather than just failing more gracefully...) But I haven't been a Windows developer for more than 10 years.

I'm pretty sure I write the file in `/tmp` because if it is written in the current working directory, it will trigger an auto-rebuild by Meteor, leading to an endless restart loop in the test.